### PR TITLE
🍭 build tweaks

### PR DESCRIPTION
**Pull request by samccone**
Minor tweaks to make it possible for others to build this locally, and ease the local dev flow :kissing_cat:

### CODE REVIEW

**Comment by samccone** [comment]
Would you like me to rebase this?

**Comment by samccone** [comment]
Would you like me to rebase this?

**Comment by justinfagnani** [comment]
Yeah, that'd be awesome

**Comment by samccone** [comment]
sure thing, 1 moment

**Comment by samccone** [comment]
kk :goat: :game_die:

File: README.md [hunk 1]
```diff
@@ -10,9 +10,9 @@ custom elements.
 polydev must be built before running. The build step externalizes inline scripts
 for CSP compliance, and copies some dependencies into convenient locations.
 
-To build, run `gulp`:
+To build, run `npm run build`:
```
Comment (justinfagnani) [inline README.md:13]: What's the benefit of this over `gulp`? I haven't quite grokked yet why a package manager should be use to build the project.

File: gulpfile.js [hunk 2]
```diff
@@ -13,6 +13,10 @@ gulp.task('lint', function() {
     .pipe(eslint.format());
 });
 
+gulp.task('dev', function() {
+  return gulp.watch('src/**/*', ['src']);
```
Comment (justinfagnani) [inline gulpfile.js]: Can you add bower_components/element-zones?

File: README.md (hunk 1 above)
Comment (samccone) [inline README.md:13]: the big reason here is
https://docs.npmjs.com/misc/scripts#path

So a user does not have to have gulp installed globally, and rather will use the version that we have specified in node_modules.

For people that work across many projects, with many versions of gulp/grunt/node/iojs depending on a single global install is quite error prone :crying_cat_face:

File: gulpfile.js (hunk 2 above)
Comment (samccone) [inline gulpfile.js]: does that change often when you are developing?

Comment (justinfagnani) [inline gulpfile.js]: Yes. Locally I have element-zones linked into my polydev client, and quite often I'm making changes to element-zones.

File: README.md (hunk 1 above)
Comment (justinfagnani) [inline README.md:13]: ok, thanks for the info. +1

File: gulpfile.js (hunk 2 above)
Comment (samccone) [inline gulpfile.js]: interesting ok :+1:

**Comment by samccone** [comment]
all set @justinfagnani :hand:

**Comment by justinfagnani** [comment]
Thanks!

**Pull request merged**